examples/scripts/ct_abel_tune.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
```

Comment:
Standard example script header is missing. I know it's also missing on the other _tune example -- mea culpa -- but I'll fix that.

Reply:
Fixed by copying from the other tuning script you added.